inspektor/commands/github.py

```diff
@@ -10,10 +10,56 @@
 # See LICENSE for more details.
 
 import logging
+import os
 
 from cliff.command import Command
 
-from inspektor.patch import GithubPatchChecker
+from inspektor import utils
+from inspektor.patch import PatchChecker
+
+
+class GithubPatchChecker(PatchChecker):
+
+    def __init__(self, args, patch=None, logger=logging.getLogger('')):
+        super(GithubPatchChecker, self).__init__(args=args, patch=patch,
+                                                 logger=logger)
+        self.github_id = args.gh_id
+        self.github_parent_project = args.parent_project
+        assert self.github_id is not None
+        assert self.github_parent_project is not None
+        self.patch = self._fetch_from_github()
+
+    def _get_github_repo_name(self):
+        return self.vcs.get_repo_name()
+
+    def _get_github_url(self):
+        """
+        Gets the correct github URL for the given project.
+        """
+        repo = self._get_github_repo_name()
+        return ("https://github.com/%s/%s/pull/%s.patch" %
```

Comment:
I'd really prefer allowing to specify `git` remote (by default `origin`) and simply fetch the change and checkout to it instead of custom handling... Anyway do you actually know of anyone still using this feature? It used to be very cool but as computers are more powerful I simply checkout each commit and run the full check, instead of using this github checker.